APIURLS.md

```diff
@@ -0,0 +1,110 @@
+        Update User Last active:
+            GET localhost:8080/API/v1/Users/id/lastActive
+            PATCH localhost:8080/API/v1/Users/id
+        User Detailed Name Card:
+            GET localhost:8080/API/v1/Users/id/fullname
+            GET localhost:8080/API/v1/Users/id/emailaddress
+            GET localhost:8080/API/v1/Users/id/publicname
+    Task Board Management:
+        Tasks Database:
+            localhost:8080/API/v1/task/id/1
+            [Post, Put, Get, Delete]
+            localhost:8080/API/v1/tasks
+        Column Search Box:
+	        GET localhost:8080/API/v1/tasks?search={keywords}
+        Task Filter:
+            GET localhost:8080/API/v1/tasks?User=id
+        Assignees:
+            GET localhost:8080/API/v1/tasks/id/assignee
+        Task number:
+            GET localhost:8080/API/v1/columns/id/tasks/count
+        Three dot actions rename:
+            PUT localhost:8080/API/v1/tasks/id/taskname
+        Three dot actions delete:
+            DELETE localhost:8080/API/v1/tasks/id
+        Task card actions copy link:
+            GET localhost:8080/API/v1/tasks/id/link
+        Task card move position:
+            PUT localhost:8080/API/v1/tasks/id/position
+        Task card actions delete:
+            DELETE localhost:8080/API/v1/tasks/id
+        Task title
+            GET localhost:8080/API/v1/tasks/id/title
+            PUT localhost:8080/API/v1/tasks/id/title
+        Task Assignee:
+            GET localhost:8080/API/v1/tasks/id/assignee
+            PUT localhost:8080/API/v1/tasks/id/assignee
+        Task Project initials:
+            GET localhost:8080/API/v1/tasks/id/initial
+            PUT localhost:8080/API/v1/tasks/id/initial
+        No.task:
+            GET localhost:8080/API/v1/tasks/id/tasknumber
+        Create new task:
+            POST localhost:8080/API/v1/tasks/id
+    Task Card Details:
+        Task Card Status:
+            Assignee: 
+                GET localhost:8080/API/v1/tasks/id/assignee
+                PUT localhost:8080/API/v1/tasks/id/assignee
+            Reporter: 
+                GET localhost:8080/API/v1/tasks/id/reporter
+                PUT localhost:8080/API/v1/tasks/id/reporter
+            Priority: 
+                GET localhost:8080/API/v1/tasks/id/priority
+                PUT localhost:8080/API/v1/tasks/id/priority
+            Status: 
+                GET localhost:8080/API/v1/tasks/id/status
+                PUT localhost:8080/API/v1/tasks/id/status
+            Due Date: 
+                GET localhost:8080/API/v1/tasks/id/duedate
+                PUT localhost:8080/API/v1/tasks/id/duedate
+            Category:
+                GET localhost:8080/API/v1/tasks/id/category
+                PUT localhost:8080/API/v1/tasks/id/category
+            TimE Tracking: 
+                GET localhost:8080/API/v1/tasks/id/timetracking
+                PUT localhost:8080/API/v1/tasks/id/timetracking
+    Task Content:

```

Comment:
Move this API here:
`
 Create new task:

            POST localhost:8080/API/v1/tasks/id

`